### ISSUE TITLE
problem in preload script with electron

### ISSUE THREAD

**Issue by gregoire78** [twurple/twurple#297]
### Code

[use case repository](https://github.com/gregoire78/electron-twurple-test)

```
git clone https://github.com/gregoire78/electron-twurple-test
yarn install
yarn start
```

### Expected behavior

I use twurple in the `src/preload.js` script of electron with @electron-forge/plugin-webpack.
After start the app, In the console of the app (`src/renderer.js`), i should see the result of `apiClient.users.getUserByName('peteur_pan')`

### Actual Behavior

I see this error message in the console

```
Uncaught (in promise) Error: _d_fischer_cross_fetch__WEBPACK_IMPORTED_MODULE_2__.Headers is not a constructor
```

If I move the usage of twurple into the renderer process (src/renderer.js) this works well ([branch 'good'](https://github.com/gregoire78/electron-twurple-test/tree/good) in the repo)

I use @electron-forge/plugin-webpack which use **webapck 5**

Thank's

### Environment

- Version: 5.0.4
- Node version: 14.15.1
- Electron version: 15.0.0
- Operating system: Linux 5.4.0-86-generic Ubuntu x86_64

**Comment by d-fischer** [comment]
This seems to be a problem in electron-forge.

I just cloned your repo, changed all imports to requires in the preload script (I don't think electron preload scripts directly support ES modules), changed all electron-forge specific constants to direct references to the source files, added the renderer module file directly to index.html and it worked:

![Success screenshot](https://user-images.githubusercontent.com/5854687/135897572-5bdbbf52-08ea-466d-891c-59b4eeb6d49d.png)

tl;dr: I reproduced it, and I fixed it by removing electron-forge.

Please report an issue over there.
